docset/winserver2022-ps/pki/New-CertificateNotificationTask.md

```diff
@@ -11,57 +11,82 @@ title: New-CertificateNotificationTask
 # New-CertificateNotificationTask
 
 ## SYNOPSIS
-Creates a new task in the Task Scheduler that will be triggered when a certificate is replaced, expired, or about to expired.
+
```

Comment:
The synopsis the only heading we _can't_ have a newline after. It breaks PlatyPS.

```suggestion
```

Reply:
🤦‍♂️ I'll update all articles and re-push. Thanks!